src/explorer/esplora.ts

```diff
@@ -249,19 +250,23 @@ async function unblindTransactionPrevoutsAndOutputs(
   return tx;
 }
 
-export function tryToUnblindOutput(
+export async function unblindOutput(
```

Comment:
so this now throws if failing to unblind, right?

Reply:
yes (it always throw an error if failing to unblind)